README.md

```diff
@@ -42,6 +41,20 @@ Get started by cloning this repository and running
 
 Experiment testing workflows by using [this plankton model from SciVision](https://sci.vision/#/model/resnet50-plankton) to extract features from images for use in similarity search, clustering, etc.
 
+### Running Jupyter notebooks
+
+The `notebooks/` directory contains Markdown (`.md`) representations of the notebooks.
+To create Jupyter notebooks (`.ipynb`), run the following command with the conda environment activated:
+
+```sh
+jupytext --sync notebooks/*
+```
+
+If you modify the contents of a notebook, run the command after closing the notebook to re-sync the `.ipynb` and `.md` representations before committing.
```

Comment:
I brought up this use of `jupytext` in the daily catchup @jmarshrossney and no opinions were forthcoming, so I'm going to call this a worthwhile contribution and try adopting it as a routine

Reply:
Thanks Jo - I appreciate it's a slightly annoying additional step. One nice thing is that it's completely trivial to change our minds and go back to tracking the `.ipynb` files down the line!